include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -611,7 +611,11 @@ class alignment_algorithm :
 
         result_value_t res{};
 
-        res.id = idx;
+        if constexpr (traits_t::output_sequence1_id)
+            res.sequence1_id = idx;
+
+        if constexpr (traits_t::output_sequence2_id)
+            res.sequence2_id = idx;
```

Comment:
Why is res.sequence1_id and res.sequence2_id set to the same value here? They are not the same, are they?

Reply:
At the moment they are the same. This has to be changed later when we actually allow the user to give the ids from outside.